From 82d54e57fdcf2e249ae6ef6c6d04b8c9c1c7dc9e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Michal=20=C5=BDidek?= <mzidek@redhat.com>
Date: Thu, 6 Oct 2016 11:26:45 +0200
Subject: [PATCH 090/100] MAN: Wrong defaults for AD provider

ldap_user_name and ldap_group_name have
different defalts then what the man page
states.

Resolves:
https://fedorahosted.org/sssd/ticket/3022

Reviewed-by: Sumit Bose <sbose@redhat.com>
---
 src/man/sssd-ldap.5.xml | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/src/man/sssd-ldap.5.xml b/src/man/sssd-ldap.5.xml
index 66b9024bcdc6faced67c4e44f9cde7caa9a5ecc8..d8e2f513312dd8efc647259d9cca8228acc31d5d 100644
--- a/src/man/sssd-ldap.5.xml
+++ b/src/man/sssd-ldap.5.xml
@@ -267,7 +267,8 @@
                             user's login name.
                         </para>
                         <para>
-                            Default: uid
+                            Default: uid (rfc2307, rfc2307bis and IPA),
+                                     sAMAccountName (AD)
                         </para>
                     </listitem>
                 </varlistentry>
@@ -847,7 +848,8 @@
                             the group name.
                         </para>
                         <para>
-                            Default: cn
+                            Default: cn (rfc2307, rfc2307bis and IPA),
+                                     sAMAccountName (AD)
                         </para>
                     </listitem>
                 </varlistentry>
-- 
2.7.4

